src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -15213,5 +15213,24 @@ class K
 
             Assert.False(semanticInfo.IsCompileTimeConstant);
         }
+
+        [Fact]
```

Comment:
No problem...I'll do that now.  Btw, as requested I did verify that with my change all tests pass locally (including my new one) but if I revert the fix, I get the original NRE in this new test case (in the same spot that caused the crash).  Being new to this, I'm not sure what the best way is to provide evidence, but attached is a screenshot of the expected failure case
![attributeargumentlambdathistest_failed](https://user-images.githubusercontent.com/1843975/28381630-9261bf2c-6c89-11e7-9800-65e41dc8a424.png)
.